dask/hashing.py

```diff
@@ -5,11 +5,30 @@
 
 
 # Timings on a largish array:
-# - CityHash is 2x faster than MurmurHash
-# - xxHash is slightly slower than CityHash
+# - xxHash3 is faster than CityHash
+# - CityHash is 2x faster than xxHash
+# - xxHash is faster than MurmurHash
 # - MurmurHash is 8x faster than SHA1
 # - SHA1 is significantly faster than all other hashlib algorithms
 
+try:
+    import xxhash  # `python -m pip install xxhash`
+except ImportError:
+    xxhash_flag = False
```

Comment:
This should maybe be called `_xxhash_flag` since it's an implementation detail of the module. Or maybe `_xxhash_available` would be more descriptive than "flag".